doc/specifier/representation-deck.adoc

```diff
@@ -0,0 +1,126 @@
+
+* `Name` : A name that helps the specifier to know what the DeckDescription is about
+* `Domain Type`: Type of the object on which it is possible to create the representation 
+* `Precondition Expression`: An expression that determine if it is possible to create the representation
+* `Title Expression`: The initial name of the representation once created
+
+### Deck style
+
+It is possible to define the *background color* of the `Deck` representation adding a `StyleDescription` in the `DeckDescription`.
+
+In addition, adding a `ConditionalStyleDescription` allows to override the style if the `Condition` expression returns `true`.
+You may add multiple `ConditionalStyleDescription`.
+
+## Lane description
+
+The `LaneDescription` is used to define the lanes.
+
+image::images/deck-lane.png[Deck lane]
+
+* `Semantic Candidates Expression`: An expression that returns one or more semantic elements. Each semantic element will be represented by a lane
+** `self` variable is the semantic element associated the `Deck`
+* `Title Expression`: An expression that defines the title of the lane
+** `self` variable is a semantic element among the list returned by the `Semantic Candidates Expression`
+* `Label Expression`: An expression that defines the label of the lane
+* `Is Collapsible Expression`: An boolean expression. If true, it is possible to collapse the lane hence hidding all the contained cards
+
+### Lane style
+
+It is possible to define the style of the lane adding a `StyleDescription` in the `LaneDescription`.
+The customizable attributes are
+
+* Font Size
+* Italic, Bold, Underline and Strike Through
+* The text color
+* The background color of the lane
+
+In addition, adding a `ConditionalStyleDescription` allows to override the style if the `Condition` expression returns `true`.
+You may add multiple `ConditionalStyleDescription`
+
+## Card description
+
+The `CardDescription` is used to define the card inside a lane.
+
+image::images/deck-card.png[Deck card]
+
+* `Semantic Candidates Expression`: An expression that returns one or more semantic elements. Each semantic element will be represented by a `Card`
+** `self` variable is the semantic element associated the `Lane`
+* `Title Expression`: An expression that defines the title of the card 
+** `self` variable is a semantic element among the list returned by the `Semantic Candidates Expression`
+* `Label Expression`: An expression that defines the label of the card
+* `Description Expression`: The description of the card
+
+### Card style
+
+As for `LaneDescription`, it is possible to define the style of the card adding a `StyleDescription` or one or more `ConditionalStyleDescription` in the `CardDescription`.
+
+## Tools
+
+Many tools can be configured for `Lane` and `Card`.
+By default, a tool does nothing.
+A tool provides access to variables usefull in the context of the tool and execute the operations associated to the tool.
```

Comment:
Typo: ...  of the tool and execute**s** the operations associated ~~to~~ **with** the tool.

Reply:
useful not usefull